pontoon/base/static/js/manage_permissions.js

```diff
@@ -0,0 +1,136 @@
+
+    var available = $(this).parents('.user.available');
+    $(this).addClass('active').siblings('a').removeClass('active');
+    available.find('li').show();
+
+    if ($(this).is('.contributors')) {
+      available.find('li:not(".contributor")').hide();
+    }
+
+    available.find('search-wrapper input').trigger('input').focus();
+  });
+
+  // While in contributors tab, search contributors only
+  $('body').on('input.search', '.menu input[type=search]', function(e) {
+    var available = $(this).parents('.user.available');
+
+    if (available.find('label a.contributors').is('.active')) {
+      available.find('li:not(".contributor")').hide();
+    }
+  });
+
+  function setArrow(element, event) {
+    var x = event.pageX - element.offset().left;
+
+    if (element.outerWidth()/2 > x) {
+      element.addClass('left');
+    } else {
+      element.removeClass('left');
+    }
+  }
+
+  // Set translators arrow direction
+  $('body')
+    .on('mouseenter', '.general .user.select.translators li', function (e) { setArrow($(this), e); })
+    .on('mousemove', '.general .user.select.translators li', function (e) { setArrow($(this), e); });
+
+  // Select users
+  $('body').on('click.pontoon', '.user.select li', function (e) {
+    var $wrapper = $(this).parents('.user.select').parent(),
+        target = $wrapper.find('.select.translators');
+
+    if ($(this).parents('.translators').length) {
+      if ($(this).is('.left') || $(this).parents('.general').length === 0) {
+        target = $wrapper.find('.select.available');
+      } else {
+        target = $wrapper.find('.select.managers');
+      }
+    }
+
+    var clone = $(this).remove();
+
+    target.find('ul').prepend(clone.removeClass('hover'));
+    $wrapper.find('.available').trigger('input').focus();
+  });
+
+  // Hide project menu on click outside
+  $('body').bind('click.main', function (e) {
+    $('#project-selector .menu').hide();
+    $('#project-selector .select').removeClass('opened');
+    $('#project-selector .menu li').removeClass('hover');
+  });
+
+  // Add project
+  $('#project-selector .menu li').click(function() {
+    var slug = $(this).data('slug'),
+        $permsForm = $(".project-locale[data-slug='" + slug + "']");
+
+    $('.project-locale:last').after($permsForm.show());
+
+    $permsForm.append(inputHidden('project-locale-' + $permsForm.data('index') + '-has_custom_translators', 1));
+
+    // Copy Translators from the General section
+    // Reverse selector order to keep presentation order (prepend)
+    $($('.permissions-groups.general .translators li').get().reverse()).each(function() {
+      $permsForm.find('.user.available li[data-id="' + $(this).data('id') + '"]').click();
+    });
+
+    // Scroll to the right project locale
+    $('html, body').animate({
+      scrollTop: $permsForm.offset().top
+    }, 500);
+
+    $(this).hide();
+
+    if ($('#project-selector .menu li:visible').length === 0) {
+      $('#project-selector').hide();
+    }
```

Comment:
Move above Copying Translators to avoid hidding selector after selection is made.